src/components/tile/Tile.js

```diff
@@ -1,9 +1,14 @@
 import React from "react";
 
-export const Tile = () => {
-  return (
-    <div className="tile-container">
-      
-    </div>
-  );
+export const Tile = (obj) => {
```

Comment:
Maybe rename obj something like 'tileAttributes'. Obj is a bit vague. 

Perhaps attributes or something that describes what they are in relation to the tile.